main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -3177,6 +3177,19 @@ object Resolver {
     case ex: NoClassDefFoundError => Result.Err(ResolutionError.UndefinedJvmClass(className, ex.getMessage, loc))
   }
 
+  /**
+    * Returns the class reflection object for the given `className`.
+    */
+  private def lookupJvmClass2(className: String, env0: ListMap[String, Resolution], loc: SourceLocation)(implicit flix: Flix): Result[Class[_], ResolutionError with Recoverable] = {
+    lookupJvmClass(className, loc) match {
```

Comment:
So, in the future this extra lookup should go away... But. that of course requires refactoring more code.

Reply:
The point being that all Java classes and interfaces must be imported.

Reply:
Yes, but wanted to do this in a soft way so I can progress on the Java import issue before breaking everything :)